src/quo/components/links/internal_link_card/community/view.cljs

```diff
@@ -9,26 +9,31 @@
     [schema.core :as schema]))
 
 (defn- description-comp
-  [description members-count active-members-count]
-  [rn/view
-   [text/text
-    {:size                :paragraph-2
-     :number-of-lines     3
-     :accessibility-label :description}
-    description]
-   [rn/view {:style style/stat-container}
-    [community-stat/view
-     {:value               members-count
-      :icon                :i/members
-      :accessibility-label :members-count
-      :style               {:margin-right 12}
-      :text-size           :paragraph-2}]
-    (when active-members-count
-      [community-stat/view
-       {:value               active-members-count
-        :icon                :i/active-members
-        :accessibility-label :active-members-count
-        :text-size           :paragraph-2}])]])
+  [description members-count active-members-count hide-members-count?]
+  (let [positive-members-count?        (pos? members-count)
+        positive-active-members-count? (pos? active-members-count)]
```

Comment:
I guess we can rename this to `has-members?` and `has-active-members?` as it's a bit clear. WDYT?

Reply:
Totally agree, Doing that rn

Reply:
Done